File: CS/VGridControl/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Windows.Forms;
using DevExpress.XtraGrid.Views.Grid;

namespace E4423
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
            BindingList<SomeList> Some = new BindingList<SomeList>();
            Some.Add(new SomeList(12, "Alex", "Some"));
            Some.Add(new SomeList(11, "Bill", "thing"));
            Some.Add(new SomeList(1, "I", "am"));
            Some.AllowNew = true;


            vGridControlDescendant1.AllowNew = true;
            vGridControlDescendant1.DataSource = Some;
            
            gridControl1.DataSource = Some;
            gridView1.OptionsView.NewItemRowPosition = NewItemRowPosition.Top;
           


        }

    }
}

File: VB/VGridControl/Form1.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Linq
Imports System.Text
Imports System.Windows.Forms
Imports DevExpress.XtraGrid.Views.Grid

Namespace E4423
	Partial Public Class Form1
		Inherits Form
		Public Sub New()
			InitializeComponent()
			Dim Some As New BindingList(Of SomeList)()
			Some.Add(New SomeList(12, "Alex", "Some"))
			Some.Add(New SomeList(11, "Bill", "thing"))
			Some.Add(New SomeList(1, "I", "am"))
			Some.AllowNew = True


			vGridControlDescendant1.AllowNew = True
			vGridControlDescendant1.DataSource = Some

			gridControl1.DataSource = Some
			gridView1.OptionsView.NewItemRowPosition = NewItemRowPosition.Top



		End Sub

	End Class
End Namespace
